commands/prop.rb

```diff
@@ -0,0 +1,55 @@
+class Command
+  def self.prop(property)
+    case property
```

Comment:
Add?
```
when 'arch_flags_override'
  puts "Use 'arch_flags_override' to override architecture specific flags."
```

Reply:
Thanks for adding documentation for this and the other properties, as well as implementing a better way to get the list of boolean properties dynamically.

Reply:
No problem.  I think we will all sleep better once this is done.